cdap-app-fabric/src/main/java/io/cdap/cdap/internal/capability/CapabilityManagementService.java

```diff
@@ -71,7 +71,8 @@ protected void doShutdown() {
   }
 
   @Override
-  protected long runTask() throws Exception {
+  //Made public for access from tests
```

Comment:
can you annotate with `@VisibleForTesting`

Reply:
Done